deps/llvm.mk

```diff
@@ -511,9 +511,17 @@ $(eval $(call LLVM_PATCH,llvm-D34078-vectorize-fdiv))
 $(eval $(call LLVM_PATCH,llvm-5.0-NVPTX-addrspaces)) # NVPTX
 $(eval $(call LLVM_PATCH,llvm-4.0.0-D37576-NVPTX-sm_70)) # NVPTX, Remove for 6.0
 $(eval $(call LLVM_PATCH,llvm-D38765-gvn_5.0)) # Remove for 6.0
-$(eval $(call LLVM_PATCH,llvm-D42262-jumpthreading-not-i1))
+$(eval $(call LLVM_PATCH,llvm-D42262-jumpthreading-not-i1)) # remove for 7.0
 $(eval $(call LLVM_PATCH,llvm-PPC-addrspaces)) # PPC
 $(eval $(call LLVM_PATCH,llvm-PR36292-5.0)) # PPC fixes #26249, remove for 6.0
+else ifeq ($(LLVM_VER_SHORT),6.0)
+#$(eval $(call LLVM_PATCH,llvm-5.0.0_threads))
```

Comment:
just fyi the comment on this one "Cygwin and openSUSE still use win32-threads mingw" hasn't been true for over a year, ref https://build.opensuse.org/request/show/444383 and https://sourceware.org/ml/cygwin/2016-11/msg00113.html. should be safe to drop this patch nowadays.

Reply:
Cheers! That is good news.